CHANGELOG.md

```diff
@@ -42,6 +42,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   template, to the commit description. Use cases include DCO Sign Off and
   Gerrit Change Id.
 
+* `format_signed_off_by_trailer(commit)` and
+  `format_gerrit_change_id_trailer(commit)` commit templates has been added.
+
```

Comment:
nit: better to add these to the "Commit trailers" docs instead? I don't think we need to describe every detail in changelog.